php/libraries/NDB_Client.class.inc

```diff
@@ -104,7 +112,8 @@ class NDB_Client
             . "default-src 'self' 'unsafe-inline'; "
             . "script-src 'self' 'unsafe-inline' 'unsafe-eval'; "
             . "font-src 'self' data:; "
-            . "img-src 'self' data:"
+            . "img-src 'self' data:; "
+            . $config_additions
```

Comment:
What happens if:
1. The $config_additions variable is blank? (Since the end of the line changed from empty to a semicolon)
2. You set the config variable to contain either additional script-src, font-src, or img-src values?